Task3/.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```

Comment:
I think it's better to have single root `.gitignore` where entire `.idea` folder is added as ignored.